test/jest.setup.js

```diff
@@ -0,0 +1,3 @@
+// broadcast-channel should not detect node environment
+// https://github.com/pubkey/broadcast-channel/blob/master/src/util.js#L61
+process[Symbol.toStringTag] = 'Process';
```

Comment:
If broadcast-channel detects node environment, there will be error if broadcastChannelName is not defined (fixed in https://github.com/okta/okta-auth-js/pull/1163):
```js
    TypeError [ERR_INVALID_ARG_TYPE]: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received undefined

      at getPaths (node_modules/broadcast-channel/dist/es5node/methods/node.js:122:63)
      at _callee16$ (node_modules/broadcast-channel/dist/es5node/methods/node.js:734:21)
      at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)
      at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:293:22)
      at Generator.next (node_modules/regenerator-runtime/runtime.js:118:21)
      at asyncGeneratorStep (node_modules/broadcast-channel/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
      at _next (node_modules/broadcast-channel/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
      at node_modules/broadcast-channel/node_modules/@babel/runtime/helpers/asyncToGenerator.js:32:7
```